api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java

```diff
@@ -168,12 +197,30 @@
 	/**
 	 * Retrieve a specific Tag based on its tagId
 	 * 
-	 * @param tagId
+	 * @param tagId The ID of the tag  to retrieve
 	 * @return the Tag which matches the specific ID
 	 */
 	@Transactional(readOnly = true)
 	@Authorized(value = { "Manage Flags", "Test Flags" }, requireAll = false)
 	public Tag getTag(Integer tagId);

```

Comment:
You need to have at least a line between methods.

Reply:
Done

Reply:
@ssmusoke when i for example look at these https://github.com/ssmusoke/openmrs-module-patientflags/blob/dc52a85d47d157a31a173045eb23c766966d6a06/api/src/main/java/org/openmrs/module/patientflags/api/FlagService.java#L205-L221 i do not see how you have done this.